### ISSUE TITLE
Missing ALT text on article images

### ISSUE THREAD

**Issue by davidsneighbour** [davidsneighbour/samui-samui.de#2]
Article images (via `figure` shortcode) are not adding any ALT attribute to the resulting image. They should though, title for the image is given in the shortcode setup.
